macosx/VDKQueue/VDKQueue.m

```diff
@@ -1,23 +1,23 @@
-//	VDKQueue.m
```

Comment:
This is third-party, so could we keep as-is?

Reply:
Reverted to the state before the PR.